src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task03.java

```diff
@@ -35,8 +35,24 @@ public static void main(String[] args) {
      * @return null/'RED'/'ORANGE'/'YELLOW'/'GREEN'
      */
     static String getColorHealthPoint(int currentHealthPoint, int maxHealthPoint) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        String colorHealthPoint = "";
+        double percentLowLevelHealthPoint = 0.25;
+        double percentMiddleLevelHealthPoint = 0.5;
+        double percentHighLevelHealthPoint = 0.75;
```

Comment:
"Violation: 'NAME_CONSTANT' was not used for constant naming
Solution: Rename constant"